build.gradle

```diff
@@ -53,11 +53,14 @@ dependencies {
   compile group: 'org.yaml', name: 'snakeyaml', version: '1.25'
   compile group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
   compile group: 'org.apache.commons', name: 'commons-text', version: '1.2'
+  compile group: 'commons-validator', name: 'commons-validator', version: '1.4.0'
 
   compile group: 'org.opencds.cqf', name: 'cql-engine', version: '1.3.12'
   compile group: 'info.cqframework', name: 'cql', version: '1.3.17'
   compile group: 'info.cqframework', name: 'model', version: '1.3.17'
   compile group: 'info.cqframework', name: 'cql-to-elm', version: '1.3.17'
+  
+  compile group: 'org.springframework', name: 'spring-web', version: '3.0.2.RELEASE'
```

Comment:
Is there a reason to use version 3.0.2.RELEASE? Based on https://mvnrepository.com/artifact/org.springframework/spring-web, this is a release from April 2010. I'm hesitant on adding a new dependency that is two major releases behind the current version.